infra/base-images/base-builder/compile_dataflow

```diff
@@ -25,7 +24,7 @@ $CXX $CXXFLAGS -fno-sanitize=all $SANITIZER_FLAGS -std=c++11 -O2 -c \
 $CXX $CXXFLAGS -fno-sanitize=all -fPIC -std=c++11 -O2 -c \
     $SRC/libfuzzer/dataflow/DataFlowCallbacks.cpp
 
-cp $WORK/libfuzzer/DataFlow*.o /usr/lib/
+ar r $LIB_FUZZING_ENGINE $WORK/libfuzzer/DataFlow*.o
```

Comment:
would the following work for you?

```bash
export LIB_FUZZING_ENGINE="/usr/lib/DataFlow.o /usr/lib/DataFlowCallbacks.o"	
```

Reply:
I'm not sure.

Having anything *but* a single file in `LIB_FUZZING_ENGINE` is an outlier, dfsan is the only thing that does that, i believe.
I also don't believe it is documented anywhere that it should work.
It would be best to just fix this, i really can't imagine static library causing issues :)

Reply:
These two objects are compiled with different flags, that might've been the issue with `.a` library

Reply:
> These two objects are compiled with different flags, that might've been the issue with `.a` library

I'm not sure how that would work. By that logic you'd also have issues linking them together into the final executable.

Reply:
> would the following work for you?
> 
> ```shell
> export LIB_FUZZING_ENGINE="/usr/lib/DataFlow.o /usr/lib/DataFlowCallbacks.o"	
> ```

Ok, i just checked, and nope, that doesn't work:
```
make[2]: *** No rule to make target '/usr/lib/DataFlow.o /usr/lib/DataFlowCallbacks.o', needed by 'fuzz/rawspeed/RawSpeedFuzzer'.  Stop.
```